docs/installation/reference-installer.mdx

```diff
@@ -111,21 +75,48 @@ operating systems.
     svg="linux" />
 </Install>
 
+## `netdata-installer.sh`
+
+The following installation methods are a wrapper around a single script,
+[`netdata-installer.sh`](https://raw.githubusercontent.com/netdata/netdata/master/netdata-installer.sh), which handles
+the installation process itself:
+
+- [One-line installer (`kickstart.sh`)](/packaging/installer/methods/kickstart.md)
+- [Pre-compiled static binary (`kickstart-static64.sh`)](/packaging/installer/methods/kickstart64.md)
```

Comment:
Strictly speaking, the static installer is not a wrapper around the `netdata-installer.sh` script. That script is used to _create_ the static install bundle, but none of that code actually gets run on the user system when they use the static installer (the same applies to the Docker images as well actually).

Reply:
Thanks for that clarification! Some people might like using the static binary in that case.